src/class-parsely.php

```diff
@@ -634,6 +634,10 @@ private function get_managed_credentials() {
 	 * @return bool Whether credentials are being managed at the platform level.
 	 */
 	private function are_credentials_managed(): bool {
+		if ( defined( 'is_managed' ) && true === constant( 'is_managed' ) ) {
+			return true;
+		}
```

Comment:
Who will define this constant?

As per my understanding this `is_managed` value will be return by platform as an array key inside the `wp_parsely_credentials` filter data.

Reply:
So when the platform sets that, it will do it like so in its code?
```
add_filter(
	'wp_parsely_credentials',
	function() {
		return array(
			'site_id'    => '',
			'api_secret' => '',
			'is_managed' => true
		);
	}
);
```
Is this an understanding, or do we have documentation that confirms the decision?

Reply:
If we don't know what the platform will do exactly, we need to sort this out first.

Reply:
OK, thanks. I'll work on a solution for that.

Reply:
Updated, I hope it's what we want.